### PR TITLE
deployment_config -> daemon_set in ex. command

### PR DESCRIPTION
**Pull request by maxwelldb**
Typo fix as described in https://bugzilla.redhat.com/show_bug.cgi?id=1648830.

### CODE REVIEW

**Comment by maxwelldb** [comment]
@anpingli Could you confirm this? 

Additionally, do you concur that this applies to versions 3.10 and 3.11 in addition to the 3.9 listed in BZ?

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -2113,7 +2113,7 @@ configuration and replace the value of the above variables with the desired
 instance:
 
 ----
-$ oc edit dc/<deployment_configuration>
+$ oc edit dc/<daemon_set>
```
Comment (anpingli) [inline install_config/aggregate_logging.adoc]: oc edit ds/<daemon_set>
ds=daemonset
dc=depploymentconfig

Comment (maxwelldb) [reply install_config/aggregate_logging.adoc]: @anpingli Oh, yikes, good catch. Thanks!

**Comment by maxwelldb** [comment]
@angpingli Is this all good, now?

**Comment by anpingli** [comment]
Looks good. It can be applies to 3.10 and v3.11

**Comment by maxwelldb** [comment]
Thanks!

**Comment by maxwelldb** [comment]
@kalexand-rh Are you able to label this correctly, assuming the time's right for that? I lack the power.

**Comment by kalexand-rh** [comment]
Yup. @maxwelldb, are you ready for me to merge this?

**Comment by maxwelldb** [comment]
@kalexand-rh Yup.

**Pull request merged**